components/tidb_query/src/batch/executors/index_scan_executor.rs

```diff
@@ -197,10 +196,10 @@ impl ScanExecutorImpl for IndexScanExecutorImpl {
                 // receiving optional time zone first.
 
                 match flag {
-                    datum::INT_FLAG => number::decode_i64(&mut val)
+                    datum::INT_FLAG => NumberDecoder::read_i64(&mut val)
```

Comment:
Because we import both `ReadBytesExt` and `NumberDecoder` here
```
use byteorder::{BigEndian, ReadBytesExt};
use codec::prelude::NumberDecoder;

ReadBytesExt::read_i64::<BigEndian>(&mut value)....
NumberDecoder::read_i64(&mut val)...
```

Reply:
Oh! Okay.